### PR TITLE
Updated the scamandrill app to work with play 2.4.x

### PR DESCRIPTION
**Pull request by joelsonoda**
These are the changes required for the scala mandrill library to work with play 2.4.x.

### CODE REVIEW

**Comment by graingert** [comment]
Nice it's PR #24

**Comment by joelsonoda** [comment]
Fortuitous

**Comment by graingert** [comment]
Can we apply these changes with a compile time macro?

**Comment by joelsonoda** [comment]
I think maintaining separate branches will provide more flexibility than trying to use and maintain a macro. It should be straightforward, for the most part, to merge from master into 2.4.x.

**Comment by graingert** [comment]
what would the library names be?

File: build.sbt [hunk 1]
```diff
@@ -28,10 +28,10 @@ libraryDependencies ++= {
     "com.typesafe.play" %% "play-ws"          % playVersion,
     "org.slf4j"         % "slf4j-api"         % "1.7.21"
   ) ++ Seq(
-    "org.scalatest"            %%  "scalatest"       % "2.2.6"     % "test->*",
```
Comment (graingert) [inline build.sbt]: does this need changing?

File: src/main/scala/io/github/scamandrill/models/MandrillMessagesRequests.scala [hunk 2]
```diff
@@ -25,7 +25,14 @@ case object MAttachmetOrImage {
   */
 case class MVars(name: String, content: JsValue)
 case object MVars extends ((String, JsValue) => MVars) {
-  implicit val writes = Json.writes[MVars]
+  implicit val writes = new Writes[MVars] {
```
Comment (graingert) [inline src/main/scala/io/github/scamandrill/models/MandrillMessagesRequests.scala]: will this change work on 2.5?

**Comment by joelsonoda** [comment]
`"io.github.scamandrill" %% "scamandrill" % "2.4.x"` or `"io.github.scamandrill" %% "scamandrill" % "2.5.x"`

**Comment by graingert** [comment]
cool, can you add some documentation to master about this?

File: src/main/scala/io/github/scamandrill/models/MandrillMessagesRequests.scala (hunk 2 above)
Comment (joelsonoda) [inline src/main/scala/io/github/scamandrill/models/MandrillMessagesRequests.scala]: Yes, this would also work with the play-json 2.5

**Comment by joelsonoda** [comment]
Will we add the documentation in master or the gh_pages?

File: build.sbt (hunk 1 above)
Comment (joelsonoda) [inline build.sbt]: I think the correct configuration to use for that dependency is "test", I'm not sure where the ->* came from (I added it apparently).

**Comment by joelsonoda** [comment]
Maybe we create a branch of gh_pages that we merge once we push artifacts to bintray?

**Comment by graingert** [comment]
sure, gh-pages it is.

**Comment by joelsonoda** [comment]
Is there anything outstanding for this pull request?

**Comment by graingert** [comment]
well it doesn't merge cleanly...

**Comment by joelsonoda** [comment]
Well, that won't do

**Comment by graingert** [comment]
You mean it's not supposed to? It's going into the 2.4.x branch

**Comment by joelsonoda** [comment]
I mean it won't do to have it unmergable...I've manually merged from 2.4.x
now.

**Pull request merged**